ecs.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+// ecs.php
+use PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff;
+use PhpCsFixer\Fixer\ArrayNotation\ArraySyntaxFixer;
+use PhpCsFixer\Fixer\ClassNotation\FinalInternalClassFixer;
+use PhpCsFixer\Fixer\Import\NoUnusedImportsFixer;
+use PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer;
+use PhpCsFixer\Fixer\Phpdoc\GeneralPhpdocAnnotationRemoveFixer;
+use PhpCsFixer\Fixer\Phpdoc\NoBlankLinesAfterPhpdocFixer;
+use PhpCsFixer\Fixer\Phpdoc\NoEmptyPhpdocFixer;
+use PhpCsFixer\Fixer\Phpdoc\NoSuperfluousPhpdocTagsFixer;
+use PhpCsFixer\Fixer\Strict\DeclareStrictTypesFixer;
+use Symplify\EasyCodingStandard\Config\ECSConfig;
+use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
+
+return static function (ECSConfig $ecsConfig): void {
+    // Parallel
+    $ecsConfig->parallel();
+
+    $ecsConfig->cacheDirectory('.ecs-cache');
+    // Paths
+    $ecsConfig->paths([
+        __DIR__ . '/src', __DIR__ . '/tests', __DIR__ . '/ecs.php'
+    ]);
+
+    // A. full sets
+    $ecsConfig->sets([SetList::PSR_12, SetList::SPACES]);
+
+    $ecsConfig->rule(NotOperatorWithSuccessorSpaceFixer::class);
+    $ecsConfig->rule(ArraySyntaxFixer::class);
+    $ecsConfig->ruleWithConfiguration(GeneralPhpdocAnnotationRemoveFixer::class, [
+        'annotations' => ['author', 'inheritdoc']
+    ]);
+    $ecsConfig->rule(NoBlankLinesAfterPhpdocFixer::class);
+    $ecsConfig->ruleWithConfiguration(NoSuperfluousPhpdocTagsFixer::class, [
+        'allow_mixed' => true
+    ]);
+    $ecsConfig->rule(NoEmptyPhpdocFixer::class);
+    $ecsConfig->rule(NoUnusedImportsFixer::class);
+    $ecsConfig->rule(DeclareStrictTypesFixer::class);
+    $ecsConfig->ruleWithConfiguration(FinalInternalClassFixer::class, [
+        'annotation_exclude' => ['@not-fix', '@internal'],
+        'annotation_include' => [],
+        'consider_absent_docblock_as_internal_class' => \true
```

Comment:
```diff
- 'consider_absent_docblock_as_internal_class' => \true
+ 'consider_absent_docblock_as_internal_class' => true
```
?

Reply:
Not sure where that came from, the more worrying thing is that we apparently lack a rule for it. Leaving this on my todo; any of your CS feedback I want to commit as a rule that is checked.